librz/core/cmd/cmd_search_rop.c

```diff
@@ -289,7 +289,7 @@ static bool parse_reg_op_const(const RzCore *core, const char *str, RzRopConstra
 RZ_API RZ_OWN RzRopSearchContext *rz_core_rop_search_context_new(RZ_NONNULL const RzCore *core, RZ_NULLABLE const char *greparg, const bool regexp,
 	const RzRopRequestMask mask, RZ_NONNULL RZ_BORROW RzCmdStateOutput *state) {
 
-	rz_return_val_if_fail(core && state, NULL);
+	rz_return_val_if_fail(core, NULL);
```

Comment:
Why? It says the sate also shouldn't be NULL

Reply:
Noted :+1:  Changed it to `RZ_NULLABLE`, as the rz-solver can [pass](https://github.com/giridharprasath/rz-solver/blob/186bf295d2d29f62696a3f9946c61850f598b677/tests/test_rop_solver.c#L106) it as NULL